include/boost/url/impl/scheme.ipp

```diff
@@ -107,6 +107,28 @@ to_string(scheme s) noexcept
     return "<unknown>";
 }
 
+std::uint16_t
+default_port(scheme s) noexcept
+{
+    switch(s)
+    {
+    case scheme::ftp:
+        return 21;
+    case scheme::http:
+    case scheme::ws:
+        return 80;
+    case scheme::https:
+    case scheme::wss:
+        return 443;
+    case scheme::file:
+    case scheme::none:
```

Comment:
why not just let the default case handle these 0s

Reply:
```
warning: enumeration values 'none', 'unknown', and 'file' not handled in switch [-Wswitch]
```

😐️